### PR TITLE
Blog app - add forms

### PR DESCRIPTION
**Pull request by Fombi-Favour**
In this activity, we:
- added a new method in the `ApplicationController` class
- added new post, new comments and new likes

### CODE REVIEW

File: app/controllers/comments_controller.rb
Comment (fobadara) [inline app/controllers/comments_controller.rb:1]: [REQUIRED] It is a requirement for you to create a `new` action for comments and not just a `create` action. Kindly add a `new` action

File: config/routes.rb [hunk 1]
```diff
@@ -9,6 +9,10 @@
   root "users#index"
 
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :new, :create, :show]
+  end
+  resources :posts do
+    resources :comments, only: [:create]
```
Comment (fobadara) [inline config/routes.rb]: [REQUIRED] Please don't forget to include routes for the new action after creating it.

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @Fombi-Favour, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

REDO

## Highlights
✅ Descriptive README
✅ Descriptive PR summary
✅ Clean code
✅ Good first attempt
✅ Good implementation of the principle of separating concerns.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/comments_controller.rb
Comment (Fombi-Favour) [reply app/controllers/comments_controller.rb:1]: Please can you kindly explain what you meant by that. I don't understand. Thank you

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @Fombi-Favour    :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making all the requested changes good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights
The new action added to comments controller ✔️ 
Routes updated accordingly ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**